src/components/common/OfficeAddresses.vue

```diff
@@ -311,7 +311,7 @@ export default class OfficeAddresses extends Vue {
           )
         }
       }
-      if (this.isBaseCompany) {
+      if (this.isBaseCompany && this.addresses.recordsOffice) {
         this.recMailingAddress = this.addresses.recordsOffice?.mailingAddress
```

Comment:
Added a check to resolve issue with loading address on first load

Reply:
We should fix that first load issue.